src/FixedDataTableCell.js

```diff
@@ -75,6 +75,11 @@ class FixedDataTableCell extends React.Component {
      */
     isRTL: PropTypes.bool,
 
+    /**
+     * Whether this cell is visible (i.e, inside the viewport) or not.
+     */
+    isVisible: PropTypes.bool.isRequired,
```

Comment:
From https://github.com/schrodinger/fixed-data-table-2/pull/658 where we expose `isVisible` cell prop to the cell renderer so that the client can easily figure out if a given cell is within the visible viewport.